test/test_braket_integration.jl

```diff
@@ -27,6 +27,83 @@ using Braket: I, name
     end
 end
 
+@testset "BraketSimulator to Braket Conversion" begin
```

Comment:
These are great but it might be good to have a couple smaller tests too, for example covering the non-empty results situation